modules/backend/widgets/Filter.php

```diff
@@ -561,6 +562,12 @@ public function addScopes(array $scopes)
         foreach ($scopes as $name => $config) {
             $scopeObj = $this->makeFilterScope($name, $config);
 
+            // Check if user has permissions to show this filter
+            $permissions = array_get($config, 'permissions');
+            if ($permissions && !BackendAuth::getUser()->hasAccess($permissions, false)) {
```

Comment:
@Samuell1 what happens if $permissions is an empty array? Should this check be `!empty($permissions)` instead?